rdr_service/dao/genomics_dao.py

```diff
@@ -5338,12 +5338,9 @@ def get_new_pipeline_members(self, *, biobank_ids: List[str], **kwargs) -> List:
                 GenomicSetMember.genomeType == config.GENOME_TYPE_ARRAY,  # sub flows default to array sample
                 GenomicSetMember.qcStatus == GenomicQcStatus.PASS,
                 GenomicSetMember.gcManifestSampleSource.ilike('whole blood'),
-                GenomicSetMember.diversionPouchSiteFlag != 1,
-                GenomicSetMember.blockResults != 1,
                 GenomicSetMember.blockResearch != 1,
                 GenomicSetMember.ignoreFlag != 1,
-                GenomicSetMember.biobankId.in_(biobank_ids),
-                GenomicSetMember.ai_an == 'N'
+                GenomicSetMember.biobankId.in_(biobank_ids)
```

Comment:
According to the ticket, the line 5336 should also be removed to remove the criteria for Suspended/Deactivated.